docs/components/radios/index.html

```diff
@@ -13,17 +13,13 @@ <h2 id="basic-radio">Basic Radio</h2>
     <div class="example" id="vue-radioDemo" v-cloak>
       <header>
         <form class="beta-hxForm">
-          <p>
-            <label for="txtLabel">Label</label>
-            <input id="txtLabel" class="hxTextCtrl" v-model="label" />
-          </p>
           <fieldset>
             <legend class="beta-hxFieldName">Options</legend>
             <hx-checkbox-control>
               <input id="chkIsChecked" type="checkbox" v-model="isChecked" />
               <label for="chkIsChecked">
                 <hx-checkbox></hx-checkbox>
-                Checked
+                Option 1 Checked
```

Comment:
update label to `With Default Selection`

Reply:
label updated